gtsam/discrete/AlgebraicDecisionTree.h

```diff
@@ -89,24 +89,24 @@ namespace gtsam {
     }
 
     /** Create from keys and vector table */
-    AlgebraicDecisionTree //
-    (const std::vector<typename Base::LabelC>& labelCs, const std::vector<double>& ys) {
-      this->root_ = Base::create(labelCs.begin(), labelCs.end(), ys.begin(),
-          ys.end());
+    AlgebraicDecisionTree(const typename Base::LabelCs& labelCs,
+                          const std::vector<double>& ys) {
+      this->root_ =
+          Base::create(labelCs.begin(), labelCs.end(), ys.begin(), ys.end());
     }
 
     /** Create from keys and string table */
-    AlgebraicDecisionTree //
-    (const std::vector<typename Base::LabelC>& labelCs, const std::string& table) {
+    AlgebraicDecisionTree(const typename Base::LabelCs& labelCs,
```

Comment:
Same! Over-enthusiastic here :-) please be careful when replacing vectors with map, because in many places order matters.